Policies/github-privacy-statement.md

```diff
@@ -25,119 +45,150 @@ If you're **just browsing the website**, we collect the same basic information t
+We **do not** share, sell, rent, or trade User Personal Information with third parties for their commercial purposes, expect where you have specifically told us to (such as by buying an integration from Marketplace).
 
-We **do** share certain aggregated, non-personally identifying information with others about how our users, collectively, use GitHub, or how our users respond to our other offerings, such as our conferences or events. For example, we may [compile statistics on the usage of open source licenses across GitHub](https://github.com/blog/1964-open-source-license-usage-on-github-com). However, we do not sell this information to advertisers or marketers.
+We **do not** host advertising on GitHub. We may occasionally embed content from third party sites, such as YouTube, and that content may include ads. While we try to minimize the amount of ads our embedded content contains, we can't always control what third parties show. Any advertisements on individual GitHub Pages or in GitHub repositories are not sponsored by, or tracked by, GitHub. 
+
+We **do not** disclose User Personal Information outside GitHub, except in the situations listed in this section or in the section below on [Compelled Disclosure](#how-we-respond-to-compelled-disclosure).
 
-We do not host advertising on GitHub. We may occasionally embed content from third party sites, such as YouTube, and that content may include ads. While we try to minimize the amount of ads our embedded content contains, we can't always control what third parties show.
+We **do** share certain aggregated, non-personally identifying information with others about how our users, collectively, use GitHub, or how our users respond to our other offerings, such as our conferences or events. For example, we may [compile statistics on the usage of open source licenses across GitHub](https://github.com/blog/1964-open-source-license-usage-on-github-com). However, we do not sell this information to advertisers or marketers.
 
-We may share User Personal Information with your permission, so we can perform services you have requested.
+We **do** share User Personal Information with a limited number of third party vendors who process it on our behalf to provide or improve our service, and who have agreed to privacy restrictions similar to our own Privacy Statement by signing data protection agreements. Our vendors perform services such as payment processing, customer support ticketing, network data transmission, and other similar services. When we transfer your data to our vendors under [Privacy Shield](/articles/github-privacy-statement/#githubs-global-privacy-practices), we remain responsible for it. While GitHub processes all User Personal Information in the United States, our third party vendors may process data outside of the United States or the European Union. If you would like to know who our third party vendors are, please see our page on [Subprocessors](/articles/github-subprocessors-and-cookies/).
```

Comment:
I't would be nice to clarify if data associated to free accounts is transfered.

Reply:
Could you explicitly clarify to which parties which kind of data is shared about which kind of accounts (for now the table lacks this information).  Data in the form unusable for that entity doesn't count, for example if the ISP only sees encrypted connections like TLS and unable to break TLS and will never have access to the keys,  I guess it's safe to assume that it doesn't get the access to the information transferred encrypted.

Reply:
Our annual Transparency Report covers what we share with government entities. We aren't able to disclose that much information in that kind of detail.

Reply:
My question is not only about government agencies, but also about third parties, like DropBox. I guess it'd be nice to know which kind of data is shared in which cases.

Reply:
I'm not sure I understand the question. Our third party subprocessors are listed, as you saw, and they receive data for the purpose listed. For example, we store corporate documents on Dropbox; we don't store user repository contents on Dropbox, because that wouldn't work very well. 😆 If you have more specific questions, drop us an email.